huggingface/pytorch/training/docker/1.11/py3/cu113/Dockerfile.trcomp.gpu

```diff
@@ -114,11 +115,16 @@ RUN pip install --no-cache-dir \
     nltk \
     rouge-score \
     soundfile \
-    sentence-transformers
+    sentence-transformers    
 
 # Numpy version gets downgraded and becomes incompatible with PyTorch wheel in the above conda command.
 RUN pip install -U numpy==1.22.2
 
+# Fixing identified CVEs
+RUN pip install -U \
+    "protobuf==3.19.5"
```

Comment:
can we use <3.20.0 to allow for future updates? this way we could incorporate a future protobuf by just rebuilding the container.